pkg/resource/db_cluster/sdk.go

```diff
@@ -334,8 +334,10 @@ func (rm *resourceManager) sdkFind(
 				f33 = append(f33, &f33elem)
 			}
 			ko.Status.EnabledCloudwatchLogsExports = f33
+			ko.Spec.EnableCloudwatchLogsExports = f33
```

Comment:
sdk.go is generated by ack code-generator, so we have to add our changes into hook to avoid overwrite by code generator. if any question about code-generator, we can have a call.

ref:  https://github.com/joaquinservetto/ack-rds-controller/pull/1